crates/agent/src/api/create_data_plane.rs

```diff
@@ -184,30 +187,23 @@ async fn do_create_data_plane(
     let draft: tables::DraftCatalog = serde_json::from_str::<models::Catalog>(&draft_str)
         .unwrap()
         .into();
-
-    let pub_id = id_generator.lock().unwrap().next();
-    let built = publisher
-        .build(
-            user_id,
-            pub_id,
-            Some(format!("publication for data-plane {base_name}")),
-            draft,
-            insert.logs_token,
-            &data_plane_name,
-        )
-        .await?;
-
-    if built.has_errors() {
-        for err in built.output.errors() {
-            tracing::error!(scope=%err.scope, err=format!("{:#}", err.error), "data-plane-template build error")
-        }
-        anyhow::bail!("data-plane-template build failed");
-    }
-
-    _ = publisher
-        .commit(built)
+    let publication = DraftPublication {
+        user_id,
+        logs_token: insert.logs_token,
+        draft,
+        dry_run: false,
+        detail: Some(format!("publication for data-plane {base_name}")),
+        // TODO: Should we verify user authz for publications to create a data plane?
```

Comment:
that's currently handled at L69 at the routine entrypoint: we require the user is an admin of `ops/`.